client/src/index.tsx

```diff
@@ -2,17 +2,18 @@ import "bootstrap/dist/css/bootstrap.min.css";
 import "./index.css";
 
 import React from "react";
-import { render } from "react-dom";
+import {render} from "react-dom";
```

Comment:
Why remove this white space? Is this after linting with the prettier extension?

Reply:
Yup, I think prettier was broken or something for me. I added a prettier config file for the front end. 

=> (pressed save and now it also just fixed formatting for other files too)

Reply:
Thanks!